src/mbgl/renderer/render_orchestrator.cpp

```diff
@@ -87,13 +73,16 @@ class RenderTreeImpl final : public RenderTree {
         return { layerRenderItems.begin(), layerRenderItems.end() };
     }
     RenderItems getSourceRenderItems() const override {
-        return { sourceRenderItems.begin(), sourceRenderItems.end() };
+        RenderItems result;
+        result.reserve(sourceRenderItems.size());
```

Comment:
Why is this preferred to previous?
It is also called from render call - Renderer::Impl::render - preferably container copy/alloc, if needed, could be moved out of render thread (orchestrator).